src/plugins/tables/tables.js

```diff
@@ -226,7 +226,7 @@ $document.on( "init.dt", function( event ) {
 		$elm.find( "th" ).each( function() {
 			var $th = $( this ),
 				label = ( $th.attr( "aria-sort" ) === "ascending" ) ? i18nText.aria.sortDescending : i18nText.aria.sortAscending;
-			if ( $th.attr( "data-orderable" ) !== "false" ) {
+			if ( ( $th.attr( "data-orderable" ) !== "false" ) && !( $th.hasClass( "sorting_disabled" ) ) ) {
```

Comment:
Question: what is the purpose of manually adding the class "`sorting_disable`" in the table head? Is it to disable the column sorting? Why we should add it since sorting functionality and the visual button is handled well by the attribute `data-orderable`?
Thank you

Reply:
DataTables adds the class `sorting_disable` automatically, it is not manually added.

Disable column sorting using DataTables API example:
`data-wb-tables='{ "ordering" : false }'>`

There are multiple ways to disable sorting/ordering with DataTables.

If only use the DataTables API config, and not using `data-orderable` in the `<th...`, the buttons get added, with `aria-controls=undefined` and shows the buttons.